CMakeLists.txt

```diff
@@ -76,13 +86,15 @@ set(CLANG_MODERNIZE_PATH "${CLANG_EXES_LOCATION}clang-modernize")
 configure_file(templates/clang.in ${CMAKE_BINARY_DIR}/scripts/clang)
 configure_file(templates/clang++.in ${CMAKE_BINARY_DIR}/scripts/clang++)
 configure_file(templates/common/__init__.py.in ${CMAKE_BINARY_DIR}/scripts/common/__init__.py)
-configure_file(scripts/compile.in ${CMAKE_BINARY_DIR}/scripts/compile)
+configure_file(templates/compile.in ${CMAKE_BINARY_DIR}/scripts/compile)
 
-include( CTest )
-include( cmake/test.cmake )
+install(TARGETS libSas DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
+install(DIRECTORY ${CMAKE_BINARY_DIR}/scripts USE_SOURCE_PERMISSIONS DESTINATION ${CMAKE_INSTALL_PREFIX} PATTERN "*.pyc" EXCLUDE)
+install(DIRECTORY ${CMAKE_BINARY_DIR}/config USE_SOURCE_PERMISSIONS DESTINATION ${CMAKE_INSTALL_PREFIX})
 
 # Remove persistent cached variables
 unset(checker_paths CACHE)
+unset(checker_relative_paths CACHE)
```

Comment:
why are we also removing all these? I remember that you needed the checker* because of the way we collect them for the registry, but why the options?